packages/nimble-components/src/spinner/styles.ts

```diff
@@ -19,6 +19,13 @@ export const styles = css`
         aspect-ratio: 1 / 1;
     }
 
+    div.overlay {
+        z-index: 1;
+        position: absolute;
+        height: inherit;
```

Comment:
We shouldn't be configuring sizes as inherit, see: https://github.com/ni/nimble/issues/1479#issuecomment-1703313419

Reply:
I changed the display of the host to be `inline-grid` so that I could easily overlap the overlay and spinner container without having to use `position: absolute` and `height: inherit`.